lib/cache.js

```diff
@@ -131,6 +201,25 @@ with --force.`)
     this.npm.output(`Index entries: ${stats.totalEntries}`)
     this.npm.output(`Finished in ${stats.runTime.total / 1000}s`)
   }
+
+  // npm cache ls [--package <spec> ...]
+  async ls (args) {
+    const cachePath = path.join(this.npm.cache, '_cacache')
+    const cacheKeys = Object.keys(await cacache.ls(cachePath))
+    if (Array.isArray(this.npm.flatOptions.package)
+      && this.npm.flatOptions.package.length > 0) {
+      // get results for each package spec specified
+      const results = new Set()
+      for (const spec of this.npm.flatOptions.package) {
+        const keySet = await searchCachePackage(cachePath, spec, cacheKeys)
+        for (const key of keySet)
+          results.add(key)
+      }
+      [...results].sort().forEach(key => this.npm.output(key))
+      return
+    }
+    cacheKeys.sort().forEach(key => this.npm.output(key))
```

Comment:
same comment regarding the sorting here, would be best to switch this to use `localeCompare`

Reply:
I updated the sorts to use locale